flink-clients/src/test/java/org/apache/flink/client/ExecutionEnvironmentTest.java

```diff
@@ -26,13 +26,22 @@
 import org.apache.flink.api.java.io.DiscardingOutputFormat;
```

Comment:
Commit + PR message should rather read: `Initialize filesystems when run in LocalStreamEnvironment`.
The issue is not that configuration was incorrectly passed to the fs but rather that plugins where not initialized at all.

Reply:
The filesystems are initialized anyways but if we do not load them through the pluginloader the configuration is not passed to them

Reply:
Hm, we should probably remove the fallback and fail hard to exactly find these kind of issues.

Reply:
Btw same question as below: have we actually initialized plugins before your fix at all? I don't think so.